README.md

```diff
@@ -88,6 +88,8 @@ following configuration options:
 - `log-raw-metric`: logs raw metrics received from the network.  Defaults to `false`.
 - `custom-headers` : a map of strings that are added to each request sent to allow for additional network routing / request inspection.
   Not required, default is empty. Example: `--custom-headers='{"region" : "us-east-1", "service" : "event-producer"}'`
+- `dynamic-headers` : similar with `custom-headers`, but the header values are extracted from metric tags matching the
```

Comment:
Do you expect dynamic-headers to override custom headers?

Reply:
This is a good point. Documenting which one supersedes the other in case of a clash is useful for users.

Reply:
- [ ] update doc
- [ ] make custom-headers override dynamic-headers

Reply:
Is there ways to set defaults if the values are missing?

Reply:
No. What's the use case in your mind?

I think if there is a sensible default value, it should be set by `custom-headers` already.

Reply:
Say I am sending build metrics but my instance is extracting `'service-name'` and I have ALB routing rules based on the existence of that header being there but now can not set a default due to the interaction between custom headers vs dynamic headers.

Reply:
I think this is in the weeds. Build metrics are low volume, no need to over-complicate it.  Remember, this is a workaround until we can scale properly.